src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/README.md

```diff
@@ -0,0 +1,65 @@
+# AWS OTel .NET SDK for Lambda
+
+This repo contains SDK to instrument Lambda handler to create incoming span.
+
+## Installation
+
+```shell
+dotnet add package OpenTelemetry.Contrib.Instrumentation.AWSLambda
+```
+
+## Configuration
+
+Add `AddLambdaConfigurations()` to `TracerProvider`.
+
+```csharp
+TracerProvider tracerProvider = Sdk.CreateTracerProviderBuilder()
+        // add other instrumentations
+        .AddLambdaConfigurations()
+        .Build();
+```
+
+## Instrumentation
+
+### Lambda Function
+
+1. Create a function with the same signature as the original Lambda function. Call
```

Comment:
I think it's better to use "wrapper" and "actual" keywords for functions. Something like "Create a wrapper function with the same signature as your actual lambda function...".

Reply:
Will add